Validation/HGCalValidation/plugins/HGCalBHValidation.cc

```diff
@@ -169,30 +163,24 @@ void HGCalBHValidation::analyze(const edm::Event& e, const edm::EventSetup&) {
       if (bh) {
         HGCScintillatorDetId cell(df.id());
         int depth = cell.layer();
-        analyzeDigi(cell, energy, bh, depth, kount);
+        if (energy > threshold_) {
+          int ring = cell.ring();
+          int phi = cell.iphi();
+          double ring1 = cell.zside() * (ring + 0.01);
+          hdi2Oc_->Fill(ring1, (phi - 0.01));
+          if (bh) {
+            hdigEn_->Fill(energy);
+            hdigOc_->Fill(ring1, (phi - 0.01));
```

Comment:
Probably safer (for possible mistakes when updating) and cleaner:
```suggestion
          double ring = cell.ring() + 0.01;
          double phi = cell.iphi() - 0.01;
          double ring1 = cell.zside() * ring;
          hdi2Oc_->Fill(ring1, phi);
          if (bh) {
            hdigEn_->Fill(energy);
            hdigOc_->Fill(ring1, phi);
```